### PR TITLE
Fix TypeError in WorkflowsAPIController.get_versions()

### PR DESCRIPTION
**Pull request by rikeshi**
Small fix for error

`
galaxy.web.framework.decorators ERROR 2021-02-15 10:36:14,965 [p:61509,w:1,m:0] [uWSGIWorker1Core2] Uncaught exception in exposed API method:
Traceback (most recent call last):
  File "lib/galaxy/web/framework/decorators.py", line 309, in decorator
    rval = func(self, trans, *args, **kwargs)
  File "lib/galaxy/webapps/galaxy/api/workflows.py", line 262, in show_versions
    stored_workflow = self.workflow_manager.get_stored_accessible_workflow(trans, workflow_id, **kwds)
TypeError: get_stored_accessible_workflow() got an unexpected keyword argument 'key'
`

when making a GET request to

`/api/workflows/{encoded_workflow_id}/versions`

### CODE REVIEW

**Comment by bgruening** [comment]
@rikeshi this seems to be a bug, can you target a stable Galaxy release. 21.01 for example or even 20.09 I the bug is also there?

**Comment by rikeshi** [comment]
It exists back in 20.01, should i target that version with a new PR then? 19.09 does not have the bug.

**Comment by davelopez** [comment]
Hi @rikeshi, welcome, and thanks for opening a PR!
Can you please elaborate a bit on the context of how did you get the error and help me reproduce it?
I tried to create a new simple workflow and also create a couple of versions of it but I didn't get any error when calling `/api/workflows/{encoded_workflow_id}/versions` neither in `dev` nor in `21.01`. Maybe some particularity of the workflow itself?

**Comment by mvdbeek** [comment]
This happens if you set `key` in the request payload. See https://github.com/galaxyproject/galaxy/issues/11375

File: lib/galaxy/webapps/galaxy/api/workflows.py [hunk 1]
```diff
@@ -258,7 +258,7 @@ def show_versions(self, trans: GalaxyWebTransaction, workflow_id, **kwds):
 
         Lists all versions of this workflow.
         """
-        stored_workflow = self.workflow_manager.get_stored_accessible_workflow(trans, workflow_id, **kwds)
+        stored_workflow = self.workflow_manager.get_stored_accessible_workflow(trans, workflow_id)
```
Comment (simonbray) [inline lib/galaxy/webapps/galaxy/api/workflows.py]:
```suggestion
        instance = util.string_as_bool(kwds.get("instance", "false"))
        stored_workflow = self.workflow_manager.get_stored_accessible_workflow(trans, workflow_id, by_stored_id=not instance)
```

Comment (simonbray) [reply lib/galaxy/webapps/galaxy/api/workflows.py]: I think this would be correct.

Comment (rikeshi) [reply lib/galaxy/webapps/galaxy/api/workflows.py]: Is this change still relevant now that this should be fixed in bioblend?

Comment (simonbray) [reply lib/galaxy/webapps/galaxy/api/workflows.py]: I think it is, the docstring documents an `instance` param, but this doesn't work currently. And any other `kwd` will also cause an error.

Comment (rikeshi) [reply lib/galaxy/webapps/galaxy/api/workflows.py]: Okay, thanks for clarifying

**Comment by mvdbeek** [comment]
Thank you @rikeshi, I will backport this to our supported releases.

**Pull request merged**